### ISSUE TITLE
Alt Attribute

### ISSUE THREAD

**Issue by jim-rosales** [leminhson98/prj-rev-bwfs-dasmoto#2]
Great job using local images and adding them using relative paths!

Remeber to include the alt attribute when adding images as attributes optimize your site for search engine results, increase its accessibility and is used as a backup in case the image cannot be loaded.

Example:
```HTML
<img src=".\image\hacksaw.jpg" alt="Hacksaw brushes">
```

https://github.com/leminhson98/prj-rev-bwfs-dasmoto/blob/740e9704bdcd91bd1184821018d04e94265f6250/Dasmoto%20project/index.html#L15
